_includes/configuration/toolbar-location.md

```diff
@@ -4,11 +4,15 @@
 
 The `toolbar_location` option is used to position the [toolbar]({{site.baseurl}}/configure/editor-appearance/#toolbar) and [menubar]({{site.baseurl}}/configure/editor-appearance/#menubar). The default behavior is to position the toolbar and menu bar above the content area. Setting this option to `bottom` positions the toolbar and menu bar below the content area. This option works in [classic editor]({{site.baseurl}}/general-configuration-guide/use-tinymce-classic/) (Iframe) mode and inline mode.
 
+Setting this option to `auto` positions the toolbar and menu bar either:
+* Above the content area. (default)
```

Comment:
"Above the content area by default" or "Above the content area if there is space" or something?

Reply:
Fixed